ts/src/renderer/phaser/classes/PhaserProjectile.ts

```diff
@@ -9,27 +9,9 @@ class PhaserProjectile extends PhaserAnimatedEntity {
 	) {
 		super(scene, entity, `projectile/${entity._stats.type}`);
 
-		const translate = entity._translate;
-		this.gameObject = scene.add.container(
-			translate.x,
-			translate.y,
-			[ this.sprite ]
-		);
-	}
-
-	protected transform (data: {
-		x: number;
-		y: number;
-		rotation: number
-	}): void {
-		this.gameObject.setPosition(data.x, data.y);
-		this.sprite.rotation = data.rotation;
-	}
+		this.gameObject = this.sprite;
 
-	protected scale (data: {
-		x: number;
-		y: number
-	}): void {
-		this.sprite.setScale(data.x, data.y);
```

Comment:
scale method logic is not implemented anywhere in super classes. it is shared with unit. maybe remove it from unit as well and move it to animated entity class?

Reply:
Yeah there isn't one. Right now we have empty super on PhaserEntity, but we don't use scale on Item or Proj yet and can't write it until we know how the game scaling will work.